src/message.rs

```diff
@@ -860,13 +860,8 @@ pub fn dc_msg_is_info(msg: &Message) -> libc::c_int {
     0
 }
 
-// TODO should return bool /rtn
-pub fn dc_msg_is_increation(msg: &Message) -> libc::c_int {
-    if chat::msgtype_has_file(msg.type_0) && msg.state == MessageState::OutPreparing {
-        1
-    } else {
-        0
-    }
+pub fn dc_msg_is_increation(msg: &Message) -> bool {
```

Comment:
Already done in PR #498, will conflict once it is merged

Reply:
We may merge this one and I'll rebase, it is probably easier.

Reply:
Thanks. That is nice.

Reply:
I have just merged #498. This line merged successfully, but there are 3 minor conflicts in other places. Rebase/merge please.